pkg/iam/iam.go

```diff
@@ -361,44 +351,45 @@ func (s *IAMService) attachInlinePolicy(roleName string, roleType string, params
 		RoleName: aws.String(roleName),
 	}
 
-	alreadyExists := false
-
 	// check if the inline policy already exists
 	o, err := s.iamClient.ListRolePolicies(i)
 	if err == nil {
 		for _, p := range o.PolicyNames {
 			if *p == policyName(s.roleType, s.clusterName) {
-				alreadyExists = true
+				_, err := s.iamClient.DeleteRolePolicy(&awsiam.DeleteRolePolicyInput{
```

Comment:
is there a way how to check if there is a diff that you need the deletion/update instead of doing it every time?

Reply:
We can use `GetRolePolicy` to read policy and do json compare somehow.